### PR TITLE
doc(RN): first draft

### PR DESCRIPTION
**Pull request by NathalieC**
add/edit all paragraphs
Missing: API removals, links to doc pages, BS numbers
Nath will add this later, after first review.

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -1,42 +1,107 @@
...
+After application descriptors, profiles are now available in Bonita Studio.
+In Community and Teamwork editions, you can edit the default profiles mapping with organization entities.
+In Efficiency and Performance editions, you can also create custom profiles, map them to the organization, and create their Bonita Portal menus if needed.
+Profiles are bundled in files of profiles, to allow to create a file of profiles per target environment / target organization.
+
+### Process display name
+It is now possible to add a display name for a process, so the development team can still use the technical "name" and provide a readable name for users. If not set, the technical name is used.
+
+### SAML logout
+SAML logout has been implemented, so that a user logging out of Bonita is also loggued out of all applications managed by the compagny Single-Sign-On system.
```
Comment (educhastenier) [inline md/release-notes.md]: ~logging~ out => logged out

File: md/release-notes.md [hunk 2]
```diff
@@ -1,42 +1,107 @@
...
+
+### New API method to protect task assignation
+As an alternative to the current API method, a new method has been developped to prevent AssignHumanTasks to overwrite assignee programmatically. A configuration file allows the switch.
```
Comment (educhastenier) [inline md/release-notes.md]: I'm not sure how to understand what has been done, when reading this. Which config file are we talking about?

Comment (NathalieC) [reply md/release-notes.md]: Anyone who helped Pablo on this should be able to answer. I personally don't know but I agree this should be part of the article.
I'll ask Nico.

File: md/release-notes.md [hunk 3]
```diff
@@ -1,30 +1,86 @@
...
+The integration of the forms/pages in Bonita (mobile) Portal or applications and the behavior of the Button widget have been improved in order to:
+ - avoid redirecting the current window when the form submitted is displayed in the (mobile) portal and in the applications (issues [BS-16774](https://bonitasoft.atlassian.net/browse/BS-16774) and [BS-16790](https://bonitasoft.atlassian.net/browse/BS-16790) )
+ - ignore the target URL on success when the form submitted is displayed in the (mobile) portal (issue [BS-16685](https://bonitasoft.atlassian.net/browse/BS-16685) ).
+In addition to those improvements, an embedded help as been added to the Button widget in the UI Designer, explaining this behavior.
+
+### User management page in Administrator Portal is all new 
+To insure the move of all Bonita Portal pages in AngularJS, the User "more" page in the Administrator Portal has been redesigned and reimplemented.
+
+### Time picker improvement
+We brought a little more consistency to time picking in our date/time picker widget
```
Comment (educhastenier) [inline md/release-notes.md]: Can you be more precise?

File: md/release-notes.md [hunk 4]
```diff
@@ -1,30 +1,86 @@
...
 
-### Living Application
+### Workspace import moved to a menu in Bonita Studio
+Importing an existing workspace into a new version of the studio is no more available in the installer: it has been moved to a new menu "File > Import > Workspace..." in the studio, and is automatically prompted on the first start of the studio.
 
-To be documented
+### Button widget and forms integration improvements
+The integration of the forms/pages in Bonita (mobile) Portal or applications and the behavior of the Button widget have been improved in order to:
+ - avoid redirecting the current window when the form submitted is displayed in the (mobile) portal and in the applications (issues [BS-16774](https://bonitasoft.atlassian.net/browse/BS-16774) and [BS-16790](https://bonitasoft.atlassian.net/browse/BS-16790) )
+ - ignore the target URL on success when the form submitted is displayed in the (mobile) portal (issue [BS-16685](https://bonitasoft.atlassian.net/browse/BS-16685) ).
```
Comment (educhastenier) [inline md/release-notes.md]: Is it acceptable to point to JIRA ticket that no-one outside Bonitasoft will have access? => dead links

File: md/release-notes.md [hunk 5]
```diff
@@ -1,30 +1,86 @@
...
+In addition to those improvements, an embedded help as been added to the Button widget in the UI Designer, explaining this behavior.
+
+### User management page in Administrator Portal is all new 
+To insure the move of all Bonita Portal pages in AngularJS, the User "more" page in the Administrator Portal has been redesigned and reimplemented.
```
Comment (educhastenier) [inline md/release-notes.md]: ~in~ => to AngularJS

Comment (educhastenier) [inline md/release-notes.md]: Why not simply call the page "User details page"?
"more" has no functional meaning, in my view :sob:

File: md/release-notes.md (hunk 3 above)
Comment (NathalieC) [reply md/release-notes.md]: OK. Yesterday I couldn't find a simple way to phrase it. But I'll try now.

File: md/release-notes.md (hunk 4 above)
Comment (NathalieC) [reply md/release-notes.md]: Yeah. Your point (added to many more of this kind) is valid. I'll remove them.

File: md/release-notes.md (hunk 5 above)
Comment (NathalieC) [reply md/release-notes.md]: Because the button is called "More".

File: md/release-notes.md (hunk 4 above)
Comment (tbouffard) [reply md/release-notes.md]: Last time I did that and I have been notified this is not something we do. So I agree with @educhastenier

File: md/release-notes.md (hunk 2 above)
Comment (nchabanoles) [reply md/release-notes.md]: This configuration file was a Spring Configuration made by Laurent I think. But I hope it is not the final fix we implemented in the product... :-/

Comment (educhastenier) [reply md/release-notes.md]: Can anyone with knowledge clarify this topic please? @laurentleseigneur maybe?

File: md/release-notes.md (hunk 4 above)
Comment (NathalieC) [reply md/release-notes.md]: good. removed

File: md/release-notes.md (hunk 5 above)
Comment (NathalieC) [reply md/release-notes.md]: fixed

**Pull request merged**